migration-util/src/test/java/edu/unc/lib/dcr/migration/content/ACLTransformationHelpersTest.java

```diff
@@ -335,7 +355,16 @@ private void setInheritFromParent(Resource bxc3Resc, boolean inherit) {
     }
 
     private void addEmbargo(Resource bxc3Resc, String embargoEndDate) {
-        bxc3Resc.addLiteral(CDRProperty.embargoUntil.getProperty(), embargoEndDate);
+        String regex = "^\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}$"; // ISO date without milliseconds
```

Comment:
Adding the milliseconds in here probably means that the actual code is never receiving the embargo without milliseconds during the test run. It should be fine to leave off adding the milliseconds here, which would also be more true to the actual data coming from boxc3.

Also, a note that you shouldn't need to create a new model to create the typed literal down below, as in my previous comment.